sdk/eventhub/Azure.Messaging.EventHubs/stress/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# right version of dotnet?
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build-env 
+
+# Copy in engineering system needed to build - TODO: cut this down to just what is needed
+COPY /sdk/eventhub/Azure.Messaging.EventHubs/ /app/sdk/eventhub/Azure.Messaging.EventHubs/
+COPY /eng /app/eng
+COPY /tools /app/tools
```

Comment:
Do these also need to be `./` instead of `/`?

Reply:
@jsquire I'm definitely in favor of common stress components, either global or language level. For this dockerfile, my initial thought would be to create an intermediate builder image, so you then you can do something like `FROM azsdkengsys.azurecr.io/stress/dotnet/base AS build-env`?

Reply:
Development could be somewhat annoying if you're iterating on both, so we may want to provide a way to build multiple dockerfiles in sequence if it starts getting iterated on heavily.

Reply:
I really like the idea of the intermediate image.